website/docs/guides/dependencies.md

```diff
@@ -36,6 +36,45 @@ For example:
 org.postgresql:postgresql:42.2.8
 ```
 
+### Excluding Transitive Dependencies
+
+To exclude a transitive dependency of dependency in a Scala CLI project, use the `exclude` parameter when specifying
+the dependency:
+
+- `exclude=org%%name` - for Scala modules
+- `exclude=org%name` - for Java modules
+
+It requires passing the organization and module name of dependency to exclude. For example, let's say you have the following Scala code:
+
+```scala title=Main.scala
+//> using dep "com.lihaoyi::pprint:0.8.1"
+object Main extends App {
+  println("Hello")
+}
+```
+
+For example, if you want to compile it with the `pprint` library but exclude its `sourcecode` dependency. You can use
+the exclude parameter is as follows:
+
+```scala title=Main.scala
+//> using dep "com.lihaoyi::pprint:0.8.1,exclude=com.lihaoyi%%sourcecode"
+object Main extends App {
+  println("Hello")
+}
+```
+
+### Dependency classifier
+
+To specify a classifier of a dependency in a Scala CLI project, use the `classifier` parameter when specifying
+the dependency:
```

Comment:
```suggestion
To specify a classifier of a dependency in a Scala CLI project, use the `classifier` parameter:
```

Reply:
It'd be good to refer to the `coursier` API doc here, to indicate that all `coursier` classifiers are supported, theoretically.
The doc doesn't list it, but we should at the very least make it clear that it's `coursier` classifiers we're using. A power user will then be able to find the relevant `coursier` API he or she needs.
https://get-coursier.io/docs/api#classifiers